`npm start` is missing from package.json

The Readme file states:

> To build a development version of the plugin and watch changes for automatic rebuild
> 
> `npm start`

But there is no 'start' script configured in the `scripts` section of the package.json file. The result is that it is not possible to run `npm start`, it results in an error: `npm ERR! missing script: start`

Is this a documentation error or is the `start` script actually missing?

--- a/README.md
+++ b/README.md
@@ -60,8 +60,9 @@ To build the production version of the plugin
 npm run build
 ```
 
-To build a development version of the plugin and watch changes for automatic rebuild
+To build a development version, change to the local directory of the block you are working on, and run `npm start` to watch for changes and automatically rebuild as you develop. 
 ```
+cd 01-basic-esnext/
 npm start
 ```
 

